stl/inc/xloctime

```diff
@@ -733,8 +733,12 @@ protected:
 
         for (_Num = 16;; _Num *= 2) { // convert into ever larger string buffer until success
             _Str.append(_Num, '\0');
-            if (0 < (_Count = _Strftime(&_Str[0], _Str.size(), _Fmt, _Pt, _Tnames._Getptr()))) {
+            _Count = _Strftime(&_Str[0], _Str.size(), _Fmt, _Pt, _Tnames._Getptr());
+            if (0 < _Count) {
                 break;
+            } else if (0 == _Count) {

```

Comment:
I think `_Count` is zero for both out of space and invalid specifier errors. You have to check `errno` to see which it was. Yet std\tests\Dev11_0836436_get_time passes for me with this code. Are there any tests with output long enough to actually exercise the resizing path?

Ditto for `unsigned short` and `wchar_t` specializations.

Reply:
Yes, it's reasonable